### PR TITLE
Blocks: update base-styles dependency

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* This PR is a split from #15157. Here, I updated the `@wordpress/base-styles` package and updated all blocks where we leveraged core colors according to recent changes in Core:
- WordPress/gutenberg#23454
- WordPress/gutenberg#23648
- WordPress/gutenberg#23048

This updates quite a few blocks to use those new colors. Here are some examples:

screenshot 2020-08-03 at 14 31 00
screenshot 2020-08-03 at 14 30 00
screenshot 2020-08-03 at 14 28 24
screenshot 2020-08-03 at 14 27 46
screenshot 2020-08-03 at 14 26 23
screenshot 2020-08-03 at 14 25 19

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* N/A

#### Testing instructions:

* Go to Posts > Add New
* Add and play with blocks, and see if you notice any weirdness.

#### Proposed changelog entry for your changes:

* Blocks: update to use latest colors defined by WordPress.

### CODE REVIEW

**Comment by kraftbj** [comment]
Needs a rebase, but looks good. Once rebased, let's see if we can get design to fast-track review so we can avoid it going stale again.

File: extensions/blocks/contact-form/editor.scss
Comment (keoshi) [inline extensions/blocks/contact-form/editor.scss:195]: I wonder if this should be:

```suggestion
		border: 1px solid rgba( 0, 0, 0, 0.4 );
```

To be consistent will other form elements, like: https://github.com/Automattic/jetpack/blob/a234115ceb8146b6e09d00f01c2630bc3b1d305f/extensions/blocks/contact-form/editor.scss#L159

**Review by keoshi — APPROVED** [review]
Tested most of the components updated in this PR and they all look good.

Left a suggestion, but it's a minor one and shouldn't block merging this.

**Review by keoshi — APPROVED** [review]
Approving these changes once again — LGTM!

**Comment by jeherve** [comment]
r212552-wpcom

**Pull request merged**
